converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java

```diff
@@ -136,19 +135,15 @@ private void setTaxProviderTaxIdOnNode(Element element, Node thisNode) {
 	 * @return array of String program name, entity type
 	 */
 	private String[] getProgramNameEntityPair(String name) {
-		String[] pairs = new String[2];
+		String[] pairs;
 		if (MIPS_INDIVIDUAL.equalsIgnoreCase(name)) {
-			pairs[0] = MIPS_PROGRAM_NAME;
-			pairs[1] = ENTITY_INDIVIDUAL;
+			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
 		} else if (MIPS_GROUP.equalsIgnoreCase(name)) {
-			pairs[0] = MIPS_PROGRAM_NAME;
-			pairs[1] = ENTITY_GROUP;
+			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_GROUP};
 		} else if (CPCPLUS.equalsIgnoreCase(name)) {
-			pairs[0] = CPCPLUS_PROGRAM_NAME;
-			pairs[1] = ENTITY_INDIVIDUAL;
+			pairs = new String[] {CPCPLUS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
 		} else {
-			pairs[0] = name.toLowerCase(); //Unknown case
-			pairs[1] = ENTITY_INDIVIDUAL;
+			pairs = new String[] {name.toLowerCase(), ENTITY_INDIVIDUAL};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1449)